packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js

```diff
@@ -21,6 +21,7 @@ export default class ProductSearchResults extends LightningElement {
     @track showRefinementBar = true;
     sortRule;
     selectedRefinements = {};
+    flag = true;
 
     // The wire adaptor to search for products when the query, sort rule or selected refinements change.
     @wire(productsByQuery, {
```

Comment:
we should use apollo-client instead here.

Reply:
@zsardone alex is implementing that in his PR though right? If @abkap02 does it here too, they'll have a merge conflict.

Reply:
@blittle yes, Alex's PR implementing that, but does this PR needs to call apollo-client instead of productsByQuery?

Reply:
I guess I'll leave it to Alex's PR then, will pull when his will be merged.

Reply:
👍 it works!